src/components/metadata/PreviewArea.js

```
@@ -10,22 +12,100 @@ const openExternalLink = (url) => {
   let content;
 
   switch (content_type) {
     case 'text':
-      content = (
-        <pre className="whitespace-pre-wrap break-all text-sm p-3 bg-gray-50 rounded-md max-h-60 overflow-y-auto">
-          {data}
-        </pre>
-      );
+      // Basic check if content might be Markdown (e.g., contains #, *, -, etc.)
+      // This is a very naive check; a more robust one or a metadata flag would be better.
+      const mightBeMarkdown = /^(#+\s|\*\s|-\s|\[.*\]\(.*\)|`{1,3})/.test(data.substring(0, 200)); // Check start of text
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)